stores/sql.go

```diff
@@ -230,6 +205,44 @@ func isSQLite(db *gorm.DB) bool {
 	}
 }
 
+func performMigrations(db *gorm.DB) error {
+	// Perform manual migrations.
```

Comment:
Do we want to perform manual migrations after we've done the auto-migrations? I think that makes more sense.